middleware/headers.js

```diff
@@ -1,12 +1,24 @@
-const frameguard = require('frameguard');
+const helmetLib = require('helmet');
 const nocache = require('./nocache');
 const compatibility = require('./compatibility');
 const compression = require('compression');
 
-const setup = (app, { disableCompression = false, trustProxy = true, publicPath='/public'} = {}) => {
-    app.disable('x-powered-by');
+const setup = (app, {
+    disableCompression = false,
+    trustProxy = true,
+    publicPath='/public',
+    helmet
+} = {}) => {
+
     app.set('trust proxy', trustProxy);
-    app.use(frameguard('sameorigin'));
+
+    if (helmet) {
+        app.use(helmetLib(helmet));
+        return;
+    }
+
+    app.disable('x-powered-by');
+    app.use(helmetLib.frameguard('sameorigin'));
     app.use(nocache.middleware({ publicPath }));
     app.use(compatibility.middleware());
     if (!disableCompression) app.use(compression());
```

Comment:
do we want the above 3 lines outside of the helmet config? or the helmet if statement to just have an `else` with frameguard and poweredby in it, as you'd probably want the cache busting, ie/edge compatibility header, and compression enabled even with a helmet config?

Reply:
Yeah, that's what I was thinking https://github.com/HMPO/hmpo-app/pull/18#discussion_r905008249 :)